### ISSUE TITLE
CI for dispatch:mocha

### ISSUE THREAD

**Issue by keyscores** [DispatchMe/meteor-mocha#10]
This package should have it's own tests and run on CI. The irony is that it's very hard to test the test-drivers. 
The easiest option may be having a blank meteor app in this repo, that has some tests, and run that app in meteor test mode. 

It would be easier if this dispatch:moch had a --grep feature. Individual tests could be run to avoid running a full suite, and only being able to check pass/fail.

**Comment by aldeed** [comment]
Yeah I was thinking about this, and I think also that having a full Meteor app with some tests in it is the only way.

**Comment by lpgeiger** [comment]
@aldeed is the ci running? Can we have link to it?
